src/helper.rs

```diff
@@ -5,8 +5,8 @@
 //
 // Refer to the project root for licensing information.
 #[cfg(feature = "service_account")]
-use crate::service_account::ServiceAccountKey;
```

Comment:
please ensure that the feature gate is still used for any dependencies on `ServiceAccountKey`, and only that - that one is not necessarily enabled because of its reliance on crypto libraries. Right now it disables the import on line 8, leading to mayhem :)

Reply:
Oooops. I'll try to fix it soon

Reply:
This should be fixed now, hopefully 😅 . Let me know if you like it / if this PR needs something else please